### PR TITLE
[#263] Use redux-actions

### PR DESCRIPTION
**Pull request by jonase**
- [ ] Test plan | Unit test | Integration test
- [ ] Copyright header
- [ ] Code formatting
- [ ] Documentation

* Less boilerplate with createAction and handleActions
* No need need the constants modules
* redux-promise (or redux-rx) relies on "standardized" actions provided by redux-actions

### CODE REVIEW

File: client/package.json [hunk 1]
```diff
@@ -21,31 +21,33 @@
...
     "babel-preset-react": "^6.11.1",
     "css-loader": "^0.23.1",
+    "d3": "^3.5.17",
     "extract-text-webpack-plugin": "^1.0.1",
     "file-loader": "^0.9.0",
     "fixed-data-table": "^0.6.1",
     "history": "^3.0.0",
     "isomorphic-fetch": "^2.2.1",
-    "keycloak-js": "1.9.8",
+    "keycloak-js": "1.9.7",
```
Comment (gabemart) [inline client/package.json]: Is this intentional?

Comment (jonase) [inline client/package.json]: yes. We're using keycloak version 1.9.7 for local development. I suppose we'll use the same version for deployments later (ping @kardan, @iperdomo).

**Comment by gabemart** [comment]
I'm finding that when I refresh on a saved dashboard in the dashboard editor (what I call "cold loading" a dashboard) some percentage of the time I get a blank "untitled dashboard"

Investigating...

**Comment by gabemart** [comment]
Turns out the problem has nothing to do with redux-actions, it is just a pre-existing rare race condition bug #310

**Comment by gabemart** [comment]
Code checked 👍 pushing two commits to this branch, please check and then merge

**Pull request merged**